_data/languages.yml

```diff
@@ -1,30 +1,1065 @@
+  family: afa
+  scripts: [ Latn ]
+  typology:
+    - SOV
+    - synthetic
+  variants: [ mt ]
+
+-
+  code: my
+  name: Burmese
+  family: sit
+  scripts: [ Mymr ]
+  typology:
+    - SOV
+    - analytic, isolating
+  variants: [ mm ]
+
+-
+  code: ne
+  name: Nepali
+  family: inc
+  scripts: [ Deva ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ np ]
+
+-
+  code: nl
+  name: Dutch
+  family: gem
+  scripts: [ Latn ]
+  typology:
+    - SVO, VSO
+    - synthetic
+  variants: [ be, nl ]
+
+-
+  code: no
+  name: Norwegian
+  family: gem
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - fusional
+  variants: [ nb, nn, no ]
+
+-
+  code: ny
+  name: Chewa
+  family: bnt
+  scripts: [ Latn ]
+  typology:
+    - SVO
+  variants:
+
+-
+  code: or
+  name: Oriya
+  family: ori
+  scripts: [ Orya ]
+  typology:
+    - SOV
+    -
+  variants: [ id ]
+
+-
+  code: pa
+  name: Punjabi
+  family: inc
+  scripts: [ Arab, Guru ]
+  typology:
+    - SOV
+    - fusional
+  variants: [ pk, in ]
+
+-
+  code: pl
+  name: Polish
+  family: pol
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - fusional, synthetic
+  variants: [ pl ]
+
+-
+  code: ps
+  name: Pashto
+  family: pus
+  scripts: [ Arab ]
+  typology:
+    - SOV
+    - fusional
+  variants: [ af ]
+
+-
+  code: pt
+  name: Portuguese
+  family: roa
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - inflected
+  variants: [ br, pt ]
+
+-
+  code: ro
+  name: Romanian
+  family: rum
+  scripts: [ Cyrl, Latn ]
+  typology:
+    - SVO
+    - synthetic
+  variants: [ md, ro ]
+
+-
+  code: ru
+  name: Russian
+  family: sla
+  scripts: [ Cyrl ]
+  typology:
+    - SVO
+    - inflected, fusional, synthetic
+  variants: [ ru, md, kz, ua ]
+
+-
+  code: rw
+  name: Kinyarwanda
+  family: bnt
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    -
+  variants: [ rw ]
+
+-
+  code: sd
+  name: Sindhi
+  family: snd
+  scripts: [ Arab, Deva, Khoj, Sind ]
+  typology:
+    - SOV
+    - inflected
+  variants:
+
+-
+  code: si
+  name: Sinhala
+  family: ine
+  scripts: [ Sinh ]
+  typology:
+    - SOV
+    - fusional
+  variants: [ lk ]
+
+-
+  code: sk
+  name: Slovak
+  family: ine
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - synthetic
+  variants: [ sk ]
+
+-
+  code: sl
+  name: Slovenian
+  family: ine
+  scripts: [ Latn ]
+  typology:
+    - SOV
+    - fusional, inflected
+  variants: [ si ]
+
+-
+  code: sm
+  name: Samoan
+  family: map
+  scripts: [ Latn ]
+  typology:
+    - VSO
+    - analytic
+  variants: [ ws ]
+
+-
+  code: sn
+  name: Shona
+  family: nic
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - agglutinative
+  variants:
+
+-
+  code: so
+  name: Somalia
+  family: afa
+  scripts: [ Arab, Latn, Osma ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ dj, et, ke, so ]
+
+-
+  code: sq
+  name: Albanian
+  family: sqj
+  scripts: [ Latn, Elba ]
+  typology:
+    - SVO
+    - fusional
+  variants: [ al ]
+
+-
+  code: sr
+  name: Serbian
+  family: ine
```

Comment:
For Russian, you put Slavic (sla)

I think it's fine to use the major Indo-European branches (Germanic, Slavic, Baltic...), rather than the top-level family.